index.js

```diff
@@ -27,6 +27,7 @@ app.post('/new-message', (req, res) => {
   // Somehow we're getting a message with { message: [knownObject], update_id: 32413}
   // We're only interested in the message object
   if (message.message)
+    // TODO: Send this to mixpanel, to confim this is still happening
     message = message.message;
```

Comment:
We should also change all the `console.log(err);` to send the messages to mixpanel, but this can be done in a separate PR.